charts/testmachinery/local-values.yaml

```
@@ -12,6 +12,14 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+global:
+  s3Configuration:
+    server:
+      ssl: false
+      endpoint: "tbd"
```

her should also rather omit it like 
```suggestion
      # endpoint: "tbd"
```
and then use the helm `required` keyword to force that valid values are set.

We should do the same for the other s3Config Values.
With that we would get early feedback about missing configuration